package.json

```diff
@@ -83,11 +83,13 @@
     "@bazel/buildifier": "7.3.1",
     "@bazel/concatjs": "patch:@bazel/concatjs@npm%3A5.8.1#~/.yarn/patches/@bazel-concatjs-npm-5.8.1-1bf81df846.patch",
     "@bazel/jasmine": "patch:@bazel/jasmine@npm%3A5.8.1#~/.yarn/patches/@bazel-jasmine-npm-5.8.1-3370fee155.patch",
+    "@bazel/rollup": "^5.8.1",
     "@bazel/runfiles": "^5.8.1",
     "@discoveryjs/json-ext": "0.6.2",
     "@inquirer/confirm": "5.0.0",
     "@inquirer/prompts": "7.0.0",
     "@listr2/prompt-adapter-inquirer": "2.0.17",
+    "@rollup/plugin-alias": "^5.1.1",
```

Comment:
This is because of the second commit that switches the bundling from esbuild to use rollup.

https://github.com/angular/angular-cli/pull/28726/commits/0f22795da492d5f331c3f4ca0d8406a796726c42